setup.cfg

```diff
@@ -1,3 +1,78 @@
+[metadata]
+name = cobra
+url = https://opencobra.github.io/cobrapy
+download_url = https://pypi.org/project/cobra
+project_urls =
+    Source Code = https://github.com/opencobra/cobrapy
+    Documentation = https://cobrapy.readthedocs.io
+    Bug Tracker = https://github.com/opencobra/cobrapy/issues
+author = The cobrapy core development team.
+author_email = cobra-pie@googlegroups.com
+maintainer = Moritz E. Beber
+maintainer_email = moritz.beber@gmail.com
+# Please consult https://pypi.org/classifiers/ for a full list.
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)
+    License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Bio-Informatics
+license = LGPL-2.0
+description = COBRApy is a package for constraint-based modeling of metabolic networks.
+long_description = file: README.rst, INSTALL.rst
+long_description_content_type = text/x-rst
+keywords =
+    metabolism
+    biology
+    constraint-based
+    linear programming
+    mixed-integer
+    optimization
+    flux-balance analysis
+    reconstruction
+
+[options]
+zip_safe = True
+install_requires =
+    depinfo
+    future
+    numpy ~= 1.13
+    optlang ~= 1.4
+    pandas ~= 1.0
+    python-libsbml-experimental == 5.18.1
+    ruamel.yaml ~= 0.16
+    six
+    swiglpk
+tests_require =
+    tox
+packages = find:
+package_dir =
+    = src
+
+[options.packages.find]
+where = src
+
+[options.package_data]
+cobra = test/data/*
+
+[options.extras_require]
+array =
+    scipy
+development =
+    black
+    bumpversion
+    isort
```

Comment:
Does isort play nice with black yet? A while ago those would conflict (bacl would overwrite isort and vice versa), but I haven't checked in a while.

Reply:
I've put the following isort configuration into the `tox.ini`:

```ini
[isort]
skip = __init__.py
line_length = 88
indent = 4
multi_line_output = 3
include_trailing_comma = true
force_grid_wrap = 0
use_parentheses = true
lines_after_imports = 2
```

which makes it compatible with black in my experience.